lms/djangoapps/courseware/features/homepage.feature

```diff
@@ -1,12 +1,12 @@
 @shard_1
 Feature: LMS.Homepage for web users
   In order to get an idea what edX is about
-  As a an anonymous web user
+  As an anonymous web user
   I want to check the information on the home page
 
-  Scenario: User can see the "Login" button
+  Scenario: User can see the "Sign in" button
```

Comment:
All instances of the "LogIn" button should be changed to "Sign In". Any error message in Logistration should refer to "Sign In" as well. 

@rlucioni and @AlasdairSwan  what is the specific spot you are seeing "LogIn"?